src/pytask_latex/collect.py

```diff
@@ -135,18 +135,22 @@ def _add_latex_dependencies_retroactively(task, session):
     # Scan the LaTeX document for included files.
     latex_dependencies = set(scan(source.path))
 
-    # Remove duplicated dependencies which have already been added by the user.
+    # Remove duplicated dependencies which have already been added by the user and those
+    # which do not exist.
```

Comment:
I think this logic will remove too many. 

I only want to remove image types if *one of them* exists. I.e., if there is a typo in the path, pytask should still raise an error. Is that the case?

Reply:
What happens is that if your latex document includes something like ``\inlcudegraphics{image}``, the scanner will return all possible candidates like ``image.png``, ``image.eps``, etc.. .

If the image will be produced by some task and it does not exist yet, I can't infer that during the collection of tasks since the DAG is built afterwards. So, in the first run, I would not match this image.

In the second run, the image likely exists on disk and is matched which could lead to a unnecessary compile step, but latexmk will only compile changed parts which reduces runtime again.

If there is a typo, the latex compiler will complain.